package.json

```diff
@@ -31,5 +35,15 @@
       "last 1 safari version"
     ]
   },
-  "devDependencies": {}
+  "devDependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
+    "@babel/plugin-proposal-object-rest-spread": "^7.9.0",
+    "@babel/preset-env": "^7.9.0",
+    "@babel/preset-flow": "^7.9.0",
+    "@babel/preset-react": "^7.9.4",
+    "@babel/preset-typescript": "^7.9.0",
+    "@expo/webpack-config": "^0.11.14",
```

Comment:
I don't think we need this, don't we?

Reply:
Shoot! It looks like I pointed 4 lines 😨 
What I mean is only the last line, `@expo/webpack-config`. I don't see we use it anywhere.